docs/source/examples/docker-containers.rst

```diff
@@ -27,27 +27,25 @@ For example, to use the :code:`ubuntu:20.04` image from Docker Hub:
   run: |
     # Will run inside container
 
-For Docker images hosted on private registries, you can provide the registry authentication details using the :code:`envs` section:
+For Docker images hosted on private registries, you can provide the registry authentication details using :ref:`task environment variables <env-vars>`:
 
 .. code-block:: yaml
 
-  # private_docker.yaml
+  # ecr_private_docker.yaml
   resources:
     image_id: docker:<your-private-image>:<tag>
```

Comment:
If you're giving a concrete example, might as well also update this to something concrete: 
```suggestion
    image_id: docker:<your-user-id>.dkr.ecr.us-east-1.amazonaws.com/your-private-image:latest
```

Reply:
Oh actually we only require users to specify `your-private-image` in the `image_id` section... See [here](https://github.com/skypilot-org/skypilot/blob/e5e400ba9b2b140866cbb8c9e3e4bb88df18dd33/sky/skylet/providers/command_runner.py#L146)

Reply:
I see the confusion. Maybe we could support both? Let me raise an PR to fix it

Reply:
Done in #2574